controllers/main.go

```diff
@@ -1646,9 +1646,11 @@ func (controller *MainController) LoginPost(c web.C, r *http.Request) (string, i
 
 	log.Infof("Login POST from %v, email %v", remoteIP, user.Email)
 
-	if user.EmailVerified == 0 {
-		session.AddFlash("You must validate your email address", "loginError")
-		return controller.Login(c, r)
+	if verified := user.EmailVerified; controller.getNetworkName() != "testnet" {
+		if verified == 0 {
```

Comment:
Why assign `user.EmailVerified` to the variable `verified` when it can be used directly here:`if user.EmailVerified == 0`?

Reply:
Also, I believe both if statements can be merged into 1.

Reply:
If statements  have been merged and, verified replaced with user.verified ..